src/EthersSafe.ts

```diff
@@ -67,7 +104,7 @@ class EthersSafe implements Safe {
    * @returns The chainId of the connected network
    */
   async getNetworkId(): Promise<number> {
```

Comment:
I would call it `getChainId` (as theoretically there is a difference between network id and chain id ... would have to look for the documentation)